sphinxcontrib/openapi/cmdline.py

```diff
@@ -0,0 +1,117 @@
+
+from __future__ import unicode_literals
+
+import argparse
+import collections
+import io
+import logging
+
+import yaml
+
+from sphinxcontrib.openapi import openapi20
+from sphinxcontrib.openapi import openapi30
+
+
+class _YamlOrderedLoader(yaml.SafeLoader):
+    pass
+
+
+_YamlOrderedLoader.add_constructor(
+    yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG,
+    lambda loader, node: collections.OrderedDict(loader.construct_pairs(node))
+)
+
+
+class OpenApi(object):
+
+    def __init__(self):
+        self.options = {}
+
+    def run(self, options):
```

Comment:
Any reason we need to bother nesting this in a class? Couldn't it just be a top-level function?

Reply:
Fixed in a new commit.